awx/api/views.py

```diff
@@ -185,7 +187,7 @@ def unattach(self, request, *args, **kwargs):
 
 class ApiRootView(APIView):
 
-    authentication_classes = []
+    # authentication_classes = []
```

Comment:
What's this about?  Something sneaky going on here that warrants a comment/explanation?

Reply:
@ryanpetrello Indeed!  There was an issue @cchurch and I found where `api` and `api/v2` endpoints in the api browser did not show the user as logged in (top right).  This was what was causing that.  I have verified that you can still access these two endpoints without being logged in, as expected.

Reply:
Gotcha - sounds like we can lose this commented code, then?

Reply:
yep.  It shall be done.